### DISCUSSION TITLE
DietPi-Software | Dashy

### DISCUSSION THREAD

**Discussion by HyperCriSiS** [MichaIng/DietPi#5494] [Software Request]
I think this would be a really useful addition to Dietpi!

Dashy is a Dashboard for your selfhosted apps and helps you organize your self-hosted services by making them accessible from a single place

https://github.com/Lissy93/dashy

Vote for this software via up button at this post.

**Comment by ANJ-Projects** [comment]
So sad there is not a dashboard yet. Heimdall is not there either.

**Reply by Joulinar** [reply]
something like Heimdall/Dashy will be integrated into our own Dashboard.

https://github.com/ravenclaw900/DietPi-Dashboard/issues/169

**Comment by surtarso** [comment]
I've been using the beta dietpi dash and its pretty awesome! but I do miss some stuff like temperature, other disks etc...
Dashy is different tho, it shows all services you have installed with a ping function that can tell you if the service is up or not which is pretty nice to have.
I would love to see Dashy aswell!
